target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/dsa.c

```diff
@@ -1026,9 +1065,9 @@ static void rtldsa_poll_counters(struct work_struct *work)
 
 		counters = &priv->ports[i].counters;
```

Comment:
you could freely remove this variable since it's not used anymore after this commit and the later introduced new locking function also retrieves `&priv->ports[i].counters.lock` for each port itself.
Also you could rename `i` to `port` for better readability but that's just a nitpick. (you also do it in your own locking function)

Reply:
Ack, completely missed this while looking through Sharadanand's code (and when then adding my own stuff on top of it).

Added a new cleanup commit for the iterator variable.